packages/client/src/components/CouncilMap.tsx

```diff
@@ -0,0 +1,23 @@
+interface CouncilMapProps {
+  zipCode: string;
+  setCouncilId: React.Dispatch<React.SetStateAction<number | null>>;
+}
+
+export const CouncilMap = ({ zipCode, setCouncilId }: CouncilMapProps) => {

```

Comment:
When create React functional components with the `const () => {}` syntax, you can type the function using the generic type `FC` and pass your props to that. This makes things a bit more explicit for the type checker in that it is now a Component and not just Component-like.

```suggestion
export const CouncilMap: FC<CouncilMapProps> = ({ zipCode, setCouncilId }) => {
```